app/uk/gov/hmrc/cdsimportsddsfrontend/controllers/model/MiscellaneousViewModel.scala

```diff
@@ -26,7 +26,8 @@ case class MiscellaneousViewModel(quotaOrderNumber: Option[String] = None,
                                   importDutyAndOtherCharges: Option[CurrencyAmount] = None,
                                   customsOffice: Option[String] = None,
                                   natureOfTransaction: Option[String] = Some("1"),
-                                  statisticalValue: Option[CurrencyAmount] = Some(CurrencyAmount("GBP", "100"))
+                                  statisticalValue: Option[CurrencyAmount] = Some(CurrencyAmount("GBP", "100")),
+                                  writeOffViewModel: Seq[WriteOffViewModel] = Seq.empty
```

Comment:
should we call this `writeOffs` instead?

Reply:
Changed